tools/test-build

```diff
@@ -72,14 +46,10 @@ do
             clean ;;
         maven)
             maven ;;
-        cpp)
-            cpp ;;
-        flake8)
-            flake ;;
         ant)
             antbuild ;;
         all)
-            clean && maven_findbugs && sphinx && antbuild;;
+            clean && maven_findbugs && antbuild;;
```

Comment:
Where is `maven_findbugs`?  I don't see a shell function by that name.  Does this need fixing too?